compose.yml

```
@@ -0,0 +1,42 @@
+services:
+
+  bootstrap:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile
+    ports:
+      - "3000:3000/tcp"
+      - "8080:8080/tcp"
+    volumes:
+      - ./testnet:/etc/nomos
+    environment:
+      - NET_NODE_KEY=${DOCKER_COMPOSE_BOOSTRAP_NET_NODE_KEY}
+
+  libp2p-node:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile
+    volumes:
+      - ./testnet:/etc/nomos
+    deploy:
+      replicas: ${DOCKER_COMPOSE_LIBP2P_REPLICAS} 
```

You can have defaults for those variables:
```suggestion
      replicas: ${DOCKER_COMPOSE_LIBP2P_REPLICAS:-1}
```
Same for the `environment` section.
https://docs.docker.com/compose/compose-file/compose-file-v3/#variable-substitution